openedx/core/djangoapps/programs/tasks.py

```diff
@@ -622,5 +621,9 @@ def update_certificate_visible_date_on_course_update(self, course_key):
         course_id=course_key
     ).values_list('user__username', flat=True)
 
+    LOGGER.info(
+        "Task update_certificate_visible_date_on_course_update resending course certificates" +
```

Comment:
Is this missing an _f_' on the first line, or do we only need that on lines with variables?

Would it be useful to also log the course_key?

Reply:
We just need it on the string with variables since they're two distinct strings (one formatted) that we're concatenating.